dht-node/src/config/tools.ts

```diff
@@ -0,0 +1,53 @@
+/** eslint-disable n/no-sync */
+import { logger } from '@/server/logger'
```

Comment:
please replace this file with a .env parser (we actually have one in our codebase `dotenv`)

The documentation for .env.devops is missing - there is no example file

Reply:
there is no example need, because it will be created as soon as it is missing.
It will be only necessary, following your requirements, as soon as a SEED is predefined for keypair-generation.